recipes/dialign-tx/build.sh

```diff
@@ -7,17 +7,17 @@ cd ./source
 # - update architecture
 # - disable SSE optimization
 # - disable MMX optimization
-sed -i.bak -e 's/^CPPFLAGS=/#CPPFLAGS=/g' Makefile
+#sed -i.bak -e 's/^CPPFLAGS=/#CPPFLAGS=/g' Makefile
 # disable explicit CC setting
-sed -i.bak -e 's/^CC=/#CC=/g' Makefile
```

Comment:
@dpryan79 eventually, this is needed, since the makefile does a hard coded `CC=gcc`

Reply:
You can override that with `make CC=$CC`.

Reply:
> You can override that with `make CC=$CC`.

everyday is a school day... thank for the hint! :+1: